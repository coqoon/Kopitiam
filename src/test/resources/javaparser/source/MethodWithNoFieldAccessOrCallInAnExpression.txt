class Foo {
  int f;
  int a () { return 10; }
  int b (int c) { return f + a() + c; }
}
