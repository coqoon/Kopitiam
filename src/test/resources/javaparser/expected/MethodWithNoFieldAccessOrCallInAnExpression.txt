class Foo {
 int f;
 int a () { return 10; }
 int b (int c) {
   int tmp_1 = this.a();
   int tmp_2 = this.f;
   return tmp_2 + tmp_1 + c;
 }
}
